addons/product/models/product_attribute_custom_value.py

```diff
@@ -23,3 +25,8 @@ def _compute_name(self):
             if record.custom_product_template_attribute_value_id.display_name:
                 name = "%s: %s" % (record.custom_product_template_attribute_value_id.display_name, name)
             record.name = name
+
+    _sql_constraints = [
+        ('custom_value_unique', 'unique(custom_product_template_attribute_value_id, res_id, res_model)',
+        "Only one Custom Value is allowed per Attribute Value per Production.")
```

Comment:
There is indeed an error in the description.
Regarding the unlink, I'm not sure why it could cause an issue. 
Previously, we had ``_sql_constraints = [
        ('sol_custom_value_unique', 'unique(custom_product_template_attribute_value_id, sale_order_line_id)', "Only one Custom Value is allowed per Attribute Value per Sales Order Line.")
    ]``
I may be wrong, but I think the constraint is the same as it now includes ``res_model`` and ``res_id`` which means that the uniqueness is the same as it used to be for Sales => uniqueness with the record (in this case ``sale.order.line``) thanks to ``res_model`` and ``res_id`` and with the ``product.attribute.custom.value``.
Was that what you meant?

Reply:
;6'ëThanks for the details explanation. I wasn't aware of the previous constraintqwbut indeed, I agree with you that it should not change the behavior and therefore it should not cause an issue.
¢€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€€°ö